back/spec/acceptance/authoring_assistance_responses_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+require 'rspec_api_documentation/dsl'
+
+resource 'AuthoringAssistanceResponse' do
+  explanation 'Authoring assistance responses are used to store gentle suggestions on how the author could improve their input, often by running LLM prompts in parallel.'
+  header 'Content-Type', 'application/json'
+
+  context 'when super admin' do
+    before { header_token_for user }
+
+    let(:user) { create(:super_admin) }
+
+    post 'web_api/v1/ideas/:idea_id/authoring_assistance_responses' do
```

Comment:
As agreed, I didn't spend time on testing all cases for the prototype. What could have been tested still otherwise:
- Non-super-admins are not authorized.
- Case when regenerate is true.
- Case when creating the first response.
- Check if created activity is logged.
- `CustomFreePromptService`.

Reply:
I think for sanity I think you should just add one other test to ensure that non-super-admins cannot access this endpoint. It's quite important that while it is experimental that there is no back door to it IMO